packages/fcl/README.md

```diff
@@ -201,6 +196,34 @@ function App() {
 
 `@onflow/fcl` is now configured to connect to the Dev Wallet. For more information about how `@onflow/fcl` interacts with Wallet providers, you can view documentation [../dev-wallet](dev wallet)
 
+## Create Additional Accounts
+This application uses a single (root) account to demonstrate the functionality of `@onflow`. If your implementation requires creating additional accounts, you can use the following transaction.
+
+```
+const response = await fcl.send([
+    fcl.transaction`
+      transaction {
+        execute {
+          let key = "${p => p.publicKey}"
+          let code = "${p => p.code}"
+          AuthAccount(publicKeys: [key.decodeHex()], code: code.decodeHex())
```

Comment:
After today i think this line will be incorrect

Reply:
@orodio Sorry about the delay, is the new syntax available for demonstration yet?